### ISSUE TITLE
Problems with "IN" Statement in kundera-neo4j

### ISSUE THREAD

**Issue by gurkerl83** [Impetus/kundera#702]
Hi,

what is the current state of IN Clause and respectively not equal statements.
The IMDBTransactionTest persists two Actor entities.

actor1.setName("Amresh");
actor2.setName("Amir");

It seems there is are huge problems translating JPA Queries into Lucene Queries because only the first query is working with Kundera 2.15. I have not tested these on other datastores, because there are unit tests in Kundera which get passed executing those queries. For Kundera-Neo4j those are not working.

```
private void persistActorTestWhereStatement(String state) {

    System.out.println("persistKnownDatabasesTest " + state);

    EntityManager em = emf.createEntityManager();

    List<Actor> ret = null;

    try {

        //working
        TypedQuery<Actor> query = em.createQuery("SELECT a FROM Actor a WHERE a.name='Amir'", Actor.class);

        //not working
//      TypedQuery<Actor> query = em.createQuery("SELECT a FROM Actor a WHERE a.name IN ('Amir', 'Amir2', 'Amir3')", Actor.class);

        //not working
//      TypedQuery<Actor> query = em.createQuery("SELECT a FROM Actor a WHERE a.name IN :names", Actor.class);
//      List<String> names = new ArrayList<String>();
//      names.add("Amir");
//      names.add("Amir2");
//      names.add("Amir3");
//      query.setParameter("names", names);

        //not working
//      TypedQuery<Actor> query = em.createQuery("SELECT a FROM Actor a WHERE a.name=<>'Amir'", Actor.class);

        ret = query.getResultList();

    } finally {
        em.close();
    }

    if(ret != null) {

        Iterator<Actor> iterator = ret.iterator();
        while (iterator.hasNext()){
            Actor item = iterator.next();
        }
    }

    else
        System.out.println("No Actor available");   
}

```

Thanks

**Comment by chhavigangwal** [comment]
@gurkerl83 

Sorry for delayed response, however in query

**Comment by chhavigangwal** [comment]
@gurkerl83

Sorry for delayed response, however "in" query is not supported with Kundera-neo4j & Lucene as of now. The team is currently working on the fix and will update you soon.

Chhavi

**Comment by chhavigangwal** [comment]
Fixed with Kundera -2.17 release.
